### PR TITLE
fix(Paprika): UXD-791 Upgrade storybook to fix build issue

### PR DESCRIPTION
**Pull request by allison-c**
### Purpose 🚀
_clear, one sentence description of primary purpose of this PR_

### Notes ✏️

Just don't just alpha version storybook 😄 

### Updates 📦
- [ ] MAJOR (breaking) change to _these packages_
- [ ] MINOR (backward compatible) change to _these packages_
- [ ] PATCH (bug fix) change to _these packages_

### Storybook 📕
http://storybooks.highbond-s3.com/paprika/your-branch-name

### Screenshots 📸
_optional but highly recommended_

### References 🔗
_relevant Jira ticket / GitHub issues_

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "cypress:ci": "run-p --race storybook cypress:run",
     "cypress:dev": "run-p --race storybook cypress:open",
     "cypress:open": "cypress open",
-    "cypress:run": "wait-on http://localhost:9009 && cypress run --browser chrome",
+    "cypress:run": "wait-on http://localhost:9009/iframe.html && cypress run --browser chrome",
```
Comment (allison-c) [inline package.json:9]: http://localhost:9009 will return a loading bar now after upgrading storybook, so have to check the iframe.html instead

Comment (nahumzs) [reply package.json:9]: kind of bummer, I missed it when storybook was a simple thing to use.

**Review by nahumzs — DISMISSED** [review]
seems ok to me...

**Comment by nahumzs** [comment]
ohhh wait, the branch name is incorrect `UXD=791-Upgrade-storybook` it should be `-` not equal can you change it?

**Comment by allison-c** [comment]
> ohhh wait, the branch name is incorrect `UXD=791-Upgrade-storybook` it should be `-` not equal can you change it?

Ok! Pushed https://github.com/acl-services/paprika/pull/914

**Pull request closed without merging**